src/Psr15MiddlewareAdapter.php

```diff
@@ -135,9 +138,9 @@ protected function getResponse(ResponseInterface $psr7Response): Response
         $response->setContent($foundationResponse->getContent());
         $response->setProtocolVersion($foundationResponse->getProtocolVersion());
         $response->setStatusCode($foundationResponse->getStatusCode());
-        $response->setCharset($foundationResponse->getCharset() ? $foundationResponse->getCharset() : '');
+        $response->setCharset($foundationResponse->getCharset() ?: '');
```

Comment:
Why do we need this if the getCharset is removed? Why is this removed?

Reply:
He's suing the shortend ternary operator `?:`

Reply:
It was just a minor code improvement suggested by my editor.

If getCharset() returns truthy, it'll use that value, otherwise empty string (saves calling the getter twice)

Reply:
Sorry, I edited my comment while you were commenting. Could you use the null coalesce operator?

Reply:
setCharset only accepts a string. I can wrap it with an if statement so we only set it if it's set.

Reply:
I mean use it like this: `$response->setCharset($foundationResponse->getCharset() ?? '');`

Reply:
Ahh, I get you now, sorry.

Pushed.

Reply:
I've seen another additional change I could do, so I'm going to merge this PR and I'm going to add another change.

Reply:
Thanks!